### PR TITLE
DOC-120: Update Corda PV page to reflect changes from v4.4 on

### PR DESCRIPTION
**Pull request by erinking-r3**
4.4 platform version updates according to [4.4 release notes](https://docs.corda.net/docs/corda-os/4.4/release-notes.html)

### CODE REVIEW

**Review by AnnaK-R3 — APPROVED** [review]

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

File: content/en/docs/corda-os/4.6/features-versions.md [hunk 1]
```diff
@@ -63,5 +63,13 @@ of Corda as well as opt in to newer features should they happen to be available
...
+|Prevent CorDapp Hosting Issue|6|4|4.4|4.4|
+|New Flow Framework APIs|7|4|4.5|4.5|
+|API Update|8|4|4.6|4.6|
+
+{{< note >}}
+We recommend a minimum network platform version of 5 for Corda 4.5 onwards; however, a network mPV of 4 may continue to work in some instances.
+{{< /note >}}
+
 
 {{< /table >}}
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.6/features-versions.md:75]: @erinking-r3  Please move the closing table tag above the note as this is causing the build to fail.

File: content/en/docs/corda-os/4.8/features-versions.md [hunk 2]
```diff
@@ -63,5 +63,16 @@ of Corda as well as opt in to newer features should they happen to be available
...
+|API Update|9|4|4.7|4.7|
+|API Update|10|4|4.8|4.8|
+
+{{< /table >}}
+
+{{< note >}}
+We recommend a minimum network platform version of 5 for Corda 4.5 onwards; however, a network mPV of 4 may continue to work in some instances.
+{{< /note >}}
 
 {{< /table >}}
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.8/features-versions.md:78]: Remove the last table tag it's redundant and is causing the build to fail

File: content/en/docs/corda-os/4.5/features-versions.md [hunk 3]
```diff
@@ -63,5 +63,11 @@ of Corda as well as opt in to newer features should they happen to be available
...
 |Signature Constraints|4|4|4.0|4.0|
 |Underlying Support for Accounts|5|4|4.3|4.3|
+|Prevent CorDapp Hosting Issue|6|4|4.4|4.4|
+|New Flow Framework APIs|7|4|4.5|4.5|
+
+{{< note >}}
+We recommend a minimum network platform version of 5 for Corda 4.5; however, a network mPV of 4 may continue to work in some instances.
+{{< /note >}}
 
 {{< /table >}}
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-os/4.5/features-versions.md:73]: @erinking-r3 Please move the closing table tag above the note as this is causing the build to fail.

**Review by ivanterziev-r3 — CHANGES_REQUESTED** [review]
Some bad placing of table tags causing the build to fail - comments added.

File: content/en/docs/corda-os/4.5/features-versions.md (hunk 3 above)
Comment (erinking-r3) [inline content/en/docs/corda-os/4.5/features-versions.md:73]:
```suggestion
|New Flow Framework APIs|7|4|4.5|4.5|
{{< /table >}}

{{< note >}}
We recommend a minimum network platform version of 5 for Corda 4.5; however, a network mPV of 4 may continue to work in some instances.
{{< /note >}}

```

File: content/en/docs/corda-os/4.6/features-versions.md (hunk 1 above)
Comment (erinking-r3) [inline content/en/docs/corda-os/4.6/features-versions.md:75]:
```suggestion
|API Update|8|4|4.6|4.6|
{{< /table >}}

{{< note >}}
We recommend a minimum network platform version of 5 for Corda 4.5 onwards; however, a network mPV of 4 may continue to work in some instances.
{{< /note >}}

```

File: content/en/docs/corda-os/4.8/features-versions.md (hunk 2 above)
Comment (erinking-r3) [inline content/en/docs/corda-os/4.8/features-versions.md:78]:
```suggestion
{{< /note >}}

```

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**